apps/studio/src/features/editing-experience/components/ComplexEditorStateDrawer.tsx

```diff
@@ -68,7 +74,13 @@ export default function ComplexEditorStateDrawer(): JSX.Element {
       <Box px="2rem" py="1rem">
         <FormBuilder />
       </Box>
-      <Box px="2rem" pb="1.5rem">
+      <Box
+        pos="sticky"
+        bottom={0}
+        bgColor="base.canvas.default"
+        boxShadow="md"
+        p="1.5rem 2rem 1.5rem 2rem"
```

Comment:
nit: I think `px` and `py` might be easier to read?

Reply:
sgtm